cli/bpmetadata/tfconfig.go

```diff
@@ -188,6 +189,43 @@ func parseBlueprintVersion(versionsFile *hcl.File, diags hcl.Diagnostics) (strin
 	return "", nil
 }
 
+// parseBlueprintProviderVersions gets the blueprint provider_versions from the provided config
+// from the provider_meta block
+func parseBlueprintProviderVersions(versionsFile *hcl.File) ([]*ProviderVersion, error) {
+	var v []*ProviderVersion
+	var diags hcl.Diagnostics
+
+	//parse out the required providers from the config
+	var hclModule tfconfig.Module
+	hclModule.RequiredProviders = make(map[string]*tfconfig.ProviderRequirement)
+	hclModuleDiag := tfconfig.LoadModuleFromFile(versionsFile, &hclModule)
+	diags = append(diags, hclModuleDiag...)
```

Comment:
Is there a need to declare diags seperately?
```suggestion
	// parse out the required providers from the config
	var hclModule tfconfig.Module
	hclModule.RequiredProviders = make(map[string]*tfconfig.ProviderRequirement)
	diags := tfconfig.LoadModuleFromFile(versionsFile, &hclModule)
```